src/GameMenu.h

```diff
@@ -31,12 +31,12 @@ class GameMenu {
     int field_110;
     bool field_111;
     int nameCursorPos;
-    std::vector<int8_t> strArr;
+    char* strArr;
 
 public:
     int xPos;
 
-    GameMenu(std::string var1, Micro* micro, GameMenu* var3, std::vector<int8_t> var4);
+    GameMenu(std::string var1, Micro* micro, GameMenu* var3, char* imputData);
```

Comment:
Typo, it should be "inputData" instead of "imputData"

Reply:
Also, does it make sense to make "inputData" nullptr by default?

Reply:
@AntonEvmenenko Fixed

Reply:
Fixed: uses default nullptr value for inputString